cmd/kill.go

```diff
@@ -64,3 +97,50 @@ func killJob(cmd *cobra.Command, args []string) {
 		}
 	}
 }
+
+func killTasks(cmd *cobra.Command, args []string) {
+	log.Infof("Killing task [Env:%s Role:%s Name:%s Instance:%s]\n", *env, *role, *name, *instances)
+
+	//Set jobKey for the tasks to be killed.
+	task := realis.NewTask().
+		Environment(*env).
+		Role(*role).
+		Name(*name)
+
+	//Check that instance number is passed
+	if instances == nil {
+		log.Fatalln("Instance number not found. Please pass a valid instance number. If you want to pass multiple instances, please pass them as comma separated integer values")
+	}
+
+	/*
+	* In the following block, we convert instance numbers, which were passed as strings, to integer values
+	* After converting them to integers, we add them to a slice of type int32.
+	 */
+	var intErr error
+	var instanceNumber int
+
+	splitString := strings.Split(*instances, ",")
+	instanceList := make([]int32, len(splitString))
+
+	for i := range instanceList {
+		splitString[i] = strings.TrimSpace(splitString[i])
+		instanceNumber, intErr = strconv.Atoi(splitString[i])
+		if intErr != nil {
+			log.Fatalln("Instance passed should be a number. Error: " + intErr.Error())
+			break
```

Comment:
I don't think we need `break` here but just in case we should put return instead.